test/index.js

```diff
@@ -233,6 +233,22 @@ test('args are passed `type: "string"` and "multiple"', function(t) {
   t.end();
 });
 
+test('args are passed `type: "boolean"` and "multiple"', function(t) {
+  const passedArgs = ['--foo', '--foo'];
+  const passedOptions = {
+    foo: {
+      type: 'boolean',
+      multiple: true,
+    },
+  };
+  const expected = { flags: { foo: true }, values: { foo: [true, true] }, positionals: [] };
+  const args = parseArgs({ args: passedArgs, options: passedOptions });
+
+  t.deepEqual(args, expected, 'the array contains multiple `true`s');
```

Comment:
I have been trying to get the test names clear rather than add assert descriptions, which are somewhat redundant following usual practice of one assertion per test.

Reply:
Updated the test name and removed the description here.

Reply:
👍